Meteorite fails silently when I use SSH git repo in smart.json

When I use the following in the smart.json in my project, it works fine:

``` js
  "meteor": {
    "git": "https://github.com/meteor/meteor.git",
    "commit": "f87f0bd52565031d3cf99a654e9adc4662a49eb9"
  }
```

When I do the following, Meteorite silently fails with an error about fibers (see below).

``` js
  "meteor": {
    "git": "git@github.com:meteor/meteor.git",
    "commit": "f87f0bd52565031d3cf99a654e9adc4662a49eb9"
  }
```

The above can be rewritten as follows and it works.

``` js
  "meteor": {
    "git": "ssh://git@github.com/meteor/meteor.git",
    "commit": "f87f0bd52565031d3cf99a654e9adc4662a49eb9"
  }
```

In other words, URI has to be used. It's fine if Meteorite doesn't support the implicit SSH as git clone does but it probably should fail gracefully.

The fiber error looks like this:

```
$ mrt

Stand back while Meteorite does its thing
smart.json changed.. installing from smart.json

Installing Meteor

  branch: git@github.com:meteor/meteor.git#master

Installing smart packages

Downloading Meteor development bundle
######################################################################## 100.0%

module.js:340
    throw err;
          ^
Error: Cannot find module 'fibers'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.<anonymous> (/Users/praus/.meteorite/meteors/git@github.com:meteor/meteor/f87f0bd52565031d3cf99a654e9adc4662a49eb9/tools/meteor.js:1:75)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.runMain (module.js:492:10)
```

--- a/README.md
+++ b/README.md
@@ -98,7 +98,7 @@ Meteorite writes to a `smart.lock` file in the app's root directory to track the
 
 ### Example `smart.json`
 
-The `meteor` property is not required: apps will depend on Meteor's master branch by default. You can specify `meteor.branch`, `meteor.tag` or `meteor.git` to use alternate branches, tags and forks respectively.
+The `meteor` property is not required: apps will depend on Meteor's master branch by default. You can specify `meteor.branch`, `meteor.tag` or `meteor.git` to use alternate branches, tags and forks respectively. Note that `meteor.git` expects an actual URL, use `ssh://git@github.com/meteor/meteor.git` instead of `git@github.com:meteor/meteor.git`.
 
 ``` json
 {

